setup.py

```diff
@@ -56,18 +56,14 @@
         'Development Status :: 3 - Alpha',
         'Intended Audience :: Developers',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.6',
+        'Programming Language :: Python :: 2 :: Only',
         'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
-        'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
         'Topic :: Security',
     ],
 
     packages=find_packages(),
     include_package_data=True,
+    python_requires='~=2.7.13',
```

Comment:
I am not sure about this one. 2.7.14 may work also, that may be true also for some 2.7.x versions

Reply:
Your project README states: "Separate Python 2.7.13 It won't work with lower Python version"
I have not tested it on anything other than my system python2 which is 2.7.14, where it correctly installs, and python3 3.6.2 where it now refuses to install.

https://www.python.org/dev/peps/pep-0440/#version-specifiers defines the format of this version specifier, which declares that it works on any minor version equal to or greater than 2.7.13